noir-projects/noir-protocol-circuits/crates/private-kernel-lib/src/private_kernel_circuit_public_inputs_composer.nr

```diff
@@ -38,10 +42,8 @@ impl PrivateKernelCircuitPublicInputsComposer {
             tx_request.tx_context,
         );
 
-        public_inputs.min_revertible_side_effect_counter = private_call_public_inputs.min_revertible_side_effect_counter;
```

Comment:
Assigning the min counter in `propagate_from_private_call` instead, which is run by both init and inner.